### PR TITLE
Add support for Brest, FR transportation stops

### PR DESCRIPTION
**Pull request by XioNoX**
Bus, tram, aerialway.

Inspired from the existing similar functions.
This will need the existing "ref:BIBUS" tags to be mass converted to "ref:FR:Bibus".

Untested, I might need some help on that front.
First Osmose contribution, don't hesitate to let me know if there are any improvements I can do.

Thanks

### CODE REVIEW

**Comment by frodrigo** [comment]
> Untested, I might need some help on that front.

I recommend you to does not setup a frontend. You can run the backed and output eg. as geojson to display the result. Or use the Jupyter Notebook.

You also need to list the new analyser in the configuration file.

**Comment by frodrigo** [comment]
I run it. I let you check the results.
[Analyser_Merge_Public_Transport_FR_bibus-france_bretagne_finistere.geojson.gz](https://github.com/osm-fr/osmose-backend/files/13879945/Analyser_Merge_Public_Transport_FR_bibus-france_bretagne_finistere.geojson.gz)

**Comment by XioNoX** [comment]
Thanks !

I tried to run it myself using docker, but kept running into the same issue :
```
osmose-backend/docker$ docker-compose --project-name france run --rm backend ./osmose_run.py --country france
[...]
lots of "   load merge_" lines
[...]
2024-01-09 21:46:12 error: Failed to load country france
ERROR: 8
```

About the GeoJson I'm not sure what to check exactly to be sure it's fully valid.

**Comment by frodrigo** [comment]
Country `france` does not exists
```
./osmose_run.py --no-clean --skip-frontend-check --country=france_bretagne_finistere --analyser=merge_public_transport_FR_bibus --result-format=geojson
```

Add `--skip-init` for rerun.

**Comment by XioNoX** [comment]
Thanks for the help ! the latest version is ready to be merged, the .geojson output looks good to me !

File: analysers/analyser_merge_public_transport_FR_bibus.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from .Analyser_Merge import Analyser_Merge_Point, GTFS, Load_XY, Conflate, Select, Mapping, SourceDataGouv
+
+
+class Analyser_Merge_Public_Transport_FR_bibus(Analyser_Merge_Point):
+    def __init__(self, config, logger = None):
+        Analyser_Merge_Point.__init__(self, config, logger)
+        place = "Bibus"
+        self.def_class_missing_official(item = 8040, id = 1, level = 3, tags = ['merge', 'public transport', 'fix:survey', 'fix:picture'],
+            title = T_('{0} stop not integrated', place))
+        self.def_class_possible_merge(item = 8041, id = 3, level = 3, tags = ['merge', 'public transport', 'fix:chair'],
```
Comment (frodrigo) [inline analysers/analyser_merge_public_transport_FR_bibus.py:33]:
```suggestion
        self.def_class_missing_official(item = 8040, id = 151, level = 3, tags = ['merge', 'public transport', 'fix:survey', 'fix:picture'],
            title = T_('{0} stop not integrated', place))
        self.def_class_possible_merge(item = 8041, id = 153, level = 3, tags = ['merge', 'public transport', 'fix:chair'],
```

**Comment by frodrigo** [comment]
Thank you. Merged, but not deployed, yet.

**Comment by frodrigo** [comment]
Is it available online
https://osmose.openstreetmap.fr/fr/issues/open?source=448174&item=xxxx

**Comment by XioNoX** [comment]
Merci ! Tout semble correct sur le frontend.

**Pull request merged**